src/tensorflow.cc

```diff
@@ -233,12 +233,10 @@ ValidateSequenceControl(
     if (!CompareDataType(input->data_type_, tensor_datatype)) {
       return TRITONSERVER_ErrorNew(
           TRITONSERVER_ERROR_INVALID_ARG,
-          std::string(
-              "unable to load model '" + model_name + "', sequence control '" +
-              tensor_name + "': the model expects TYPE_" +
-              TRITONSERVER_DataTypeString(ConvertDataType(input->data_type_)) +
-              " but the model configuration specifies data-type " +
-              tensor_datatype)
+          (std::string("unable to load model '") + model_name +
+           "', configuration expects datatype " + tensor_datatype +
+           " for sequence control '" + tensor_name + "', model provides TYPE_" +
+           TRITONSERVER_DataTypeString(ConvertDataType(input->data_type_)))
```

Comment:
Same as the other comment in ORT backend.

Reply:
And the changes below are also unnecessary.

Reply:
Will fix them. Thanks

Reply:
The changes below were made the keep the error message symmetric between the backends. I thought it might be useful to use the same convention